### PR TITLE
add automatic self-healing when using slice plugin to t3c

### PR DESCRIPTION
**Pull request by jpappa200**
When using the slice plugin you have to add the `--consider-ims` parameter to the remap line in remap.config. This change will make it automatically add the parameter without using a delivery service parameter.  

<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Control Cache Config (`t3c`, formerly ORT)

## PR submission checklist
- [x] This PR has tests 
- [ ] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: lib/go-atscfg/remapdotconfig_test.go [hunk 1]
```diff
@@ -1599,16 +1599,16 @@ func TestMakeRemapDotConfigMidSlicePluginRangeRequestHandling(t *testing.T) {
 	ds.Active = util.BoolPtr(true)
 
 	ds2 := DeliveryService{}
-	ds2.ID = util.IntPtr(48)
+	ds2.ID = util.IntPtr(50)
 	dsType2 := tc.DSType("HTTP_LIVE_NATNL")
 	ds2.Type = &dsType2
-	ds2.OrgServerFQDN = util.StrPtr("origin.example.test")
+	//ds2.OrgServerFQDN = util.StrPtr("origin2.example.test")
```
Comment (ocket8888) [inline lib/go-atscfg/remapdotconfig_test.go:1605]: Why'd you comment this out?

Comment (jpappa200) [reply lib/go-atscfg/remapdotconfig_test.go]: probably testing and committed by accident. fixed.

**Review by traeak — APPROVED** [review]
with requested changes these look good now.

File: lib/go-atscfg/remapdotconfig_test.go [hunk 2]
```diff
@@ -1687,6 +1691,9 @@ func TestMakeRemapDotConfigMidSlicePluginRangeRequestHandling(t *testing.T) {
 
 	txt = strings.TrimSpace(txt)
 
+	if !strings.Contains(txt, "@pparam=--consider-ims") {
+		t.Fatalf("expected '--consider-ims' param with 'cache_range_requests.so' when using slice plugin to enable self healing, actual: %s", txt)
```
[the remark this answers is not in the document]
Comment (jpappa200) [reply lib/go-atscfg/remapdotconfig_test.go]: fixed

**Review by traeak — APPROVED** [review]

**Pull request merged**